src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random random = new Random();
```

Comment:
let's not make it static, because it does not quite fit in the dependency injection principle, for now, we will initialize similar objects like this, and only constants with types of primitives or wrappers will be static